Odata-docs/client/v6/getting-started.md

```diff
@@ -0,0 +1,140 @@
+
+## Using the generated code
+
+Now open your `Program.cs` file and add the following `using` statement at the top:
+
+```c#
+using Microsoft.OData.SampleService.Models.TripPin
+```
+
+By default, the OData Connected Service generates the relevant classes in the same namespace defined in the OData metadata document. In this case it's `Microsoft.OData.SampleService.Models.TripPin`.
+
+Let's create a new method in `Program` class called `ListPeople()` with the following code:
+
+```c#
+async Task ListPeople()
+{
+    var serviceRoot = "https://services.odata.org/V4/TripPinServiceRW/"
+    var context = new DefaultContainer(new Uri(serviceRoot));
+
+    IEnumerable<Person> people = await context.People.ExecuteAsync();
+    foreach (var person in people)
+    {
+        Console.WriteLine("{0} {1}", person.FirstName, person.LastName);
+    }
+}
+```
+
+The above method creates a new instance of `DefaultContainer` pointed to the root
+endpoint of the OData service we used to generate the client.
+The `DefaultContainer` class is generated by the OData Connected Service,
+it inherits from the [DataServiceContext](https://docs.microsoft.com/en-us/dotnet/api/microsoft.odata.client.dataservicecontext)
+class that's defined in the OData client library (`Microsoft.OData.Client`)
+The `DataServiceContext` keeps track of te objects created
+and fetched from the interactions with the service and allows you to
+execute queries and other operations against the service.
+
+`DefaultContainer` is generated based on the Trip Pin service and knows about
+its data model. In this case `context.People` is generated from the People
+entity set defined in the model. `context.People` is an instance of
+[DataServiceQuery&lt;Person&gt;](https://docs.microsoft.com/en-us/dotnet/api/microsoft.odata.client.dataservicequery-1).
+The `DataServiceQuery<TElement>` class allows you to execute LINQ-enabled queries against
+a particular endpoint of the service. `context.People` executes queries that return a
+collection of `Person` instances.
+The `Person` class is in turn generated based on the `Person` entity in the
+OData model and contains the properties defined in the data model.
+The generated code also includes enums, and methods corresponding to bound and unbound functions and actions.
+
+Now let's call the `ListPeople()` method from the `Main` method by adding the following
+statement in `Main`:
+```c#
+ListPeople().Wait();
+```
+Don't forget to add the following using statements as well:
+```c#
+using System.Collections.Generic;
+using System.Threading.Tasks;
+```
+
+Now the entire program should look like:
+```c#
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.OData.SampleService.Models.TripPin;
+
+namespace ODataClientExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ListPeople().Wait();
```

Comment:
You can switch main to async so as to use await or leave a comment for why you are using Wait so that new people to the language don't pick this up and run with it.